rosbag2_compression/src/rosbag2_compression/compression_factory_impl.hpp

```diff
@@ -29,12 +29,17 @@ namespace
 
 constexpr const char kCompressionFormatZstd[] = "zstd";
 
-/// Convert a string to lowercase
-std::string to_lower(const std::string & text)
+/// Verify whether two case-insensitive chars are equal
+bool compare_char(const char & c1, const char & c2)
```

Comment:
Does this work with `const char` instead of `const char &` ?

Reply:
It does